## SC3251 - Module 2
**Name:** Hussary, Elias   
**Assignment:** SC3251 - Module 2  
**Date:** 2018-05-23

---

In [22]:
%config IPCompleter.greedy=True # for better intellisense

## Import necessary modules for this notebook.
from matplotlib import pyplot as plt, rcParams
import pandas as pd
import numpy as np
from fractions import Fraction

rcParams.update({'font.size': 16, 'figure.figsize': (20,10)}) # global formatting

### Assignment Instructions
Exercises 2.28, 2.34, 2.40 from 'OpenIntro Statistics' textbook.

Bonus (3 points). Model Monty Hall problem to find chance to win. Present results graphically.

### 2.28 - Socks in a drawer.
In your sock drawer you have 4 blue, 5 gray, and 3 black socks. Half asleep one morning you grab 2 socks at random and put them on. 
Find the probability you end up wearing;
* (a) 2 blue socks 
* (b) no gray socks 
* (c) at least 1 black sock 
* (d) a green sock 
* (e) matching socks

In [49]:
# declare variables for 2.28
blue_socks = 4
gray_socks = 5
black_socks = 3
total_socks = blue_socks + gray_socks + black_socks
f'Total Socks = {total_socks}'

'Total Socks = 12'

In [48]:
# a) blue socks
p_first_blue = 4 / 12
p_second_blue = 3 / 11

p_two_blue = p_first_blue * p_second_blue
f'P = {p_two_blue:.2%}'

'P = 9.09%'

\begin{equation*}
P(Blue1, Blue2)   = \frac{4}{12} \times \frac{3}{11} = \frac{12}{132} = 9.09\%
\end{equation*}

a) The probability of wearing two blue socks is **9.09%**.

In [52]:
# b) no gray socks
p_first_no_gray = 7 / 12
p_second_no_gray = 6 / 11

p_no_gray = p_first_no_gray * p_second_no_gray
f'P = {p_no_gray:.2%}'

'P = 31.82%'

\begin{equation*}
P(NoGray1, NoGray2)   = \frac{7}{12} \times \frac{6}{11} = \frac{42}{132} = 31.82\%
\end{equation*}

a) The probability of wearing no gray socks is **31.82%**.

### 2.34 - Ace of clubs wins.
Consider the following card game with a well-shuffled deck of cards. If you draw a red card, you win nothing. 
If you get a spade, you win \$5. For any club, you win \$10 plus an extra \$20 for the ace of clubs. 
* (a) Create a probability model for the amount you win at this game. Also, find the expected winnings for a single game and the standard deviation of the winnings. 
* (b) What is the maximum amount you would be willing to pay to play this game? Explain your reasoning.

### 2.40 - European roulette. 
The game of European roulette involves spinning a wheel with 37 slots: 18 red, 18 black, and 1 green. A ball is spun onto the wheel and will eventually land in a slot, where each slot has an equal chance of capturing the ball. Gamblers can place bets on red or black. If the ball lands on their color, they double their money. If it lands on another color, they lose their money.
* (a) Suppose you play roulette and bet \$3 on a single round. What is the expected value and standard deviation of your total winnings?   
* (b) Suppose you bet \$1 in three different rounds. What is the expected value and standard deviation of your total winnings? 
* (c) How do your answers to parts (a) and (b) compare? What does this say about the riskiness of the two games?